proto/desmos/profiles/v3/models_chain_links.proto

```diff
@@ -115,28 +119,63 @@ message HexAddress {
   string prefix = 2 [ (gogoproto.moretags) = "yaml:\"prefix\"" ];
 }
 
-// SingleSignatureData is the signature data for a single signer
-message SingleSignatureData {
+// --------------------------------------------------------------------------------------------------------------------
+
+// SingleSignature is the signature data for a single signer
+message SingleSignature {
   option (gogoproto.goproto_getters) = false;
   option (gogoproto.equal) = true;
-  option (cosmos_proto.implements_interface) = "SignatureData";
+  option (cosmos_proto.implements_interface) = "CosmosSignature";
 
-  // Mode is the signing mode of the single signer
-  cosmos.tx.signing.v1beta1.SignMode mode = 1;
+  // ValueEncoding represents how the value has been encoded before being signed
+  SignatureValueEncoding value_encoding = 1
```

Comment:
I would remain the naming `mode` with `SignMode` instead of `value_encoding` with `SignatureValueEncoding`.

Reply:
I thought about this too, but then I thought: _signature modes are things like Secp256k1, Ed25519, etc while in this case we want to specify how the signature value has been encoded before signing (Amino, Protobuf, PersonalSign, etc). So I think this makes more sense_. Does it make sense to you?

Reply:
I think `PersonalSign` is not a encoding method since the plain text of it is raw text, it is the method to sign a raw text message into signature. Besides, `secp256k1` and `ed25519` algorithms are based on the keys in the [code base](https://github.com/cosmos/cosmos-sdk/tree/6f070623741fe0d6851d79ada41e6e2b1c67e236/proto/cosmos/crypto), since they use different curves to sign message with the key.

Reply:
Maybe we can just call it `SignatureType` then? And have `SIGNATURE_TYPE_COSMOS_AMINO`, `SIGNATURE_TYPE_EVM_PERSONAL_SIGN`, etc?

Reply:
@RiccardoM That is great! I agree with it.

Reply:
I ended up going to `SignatureValueType` which I think it's more explanatory than `SignatureType` which might be confused for something else